src/test/scala/guessbook/GuessBookSpec.scala

```diff
@@ -0,0 +1,130 @@
+package guessbook
+
+import guesswho.GenderEnum
+import org.scalatest.wordspec.AnyWordSpec
+
+class GuessBookSpec extends AnyWordSpec {
+  private val game = GuessBook
+
+  "guessBook" should {
+    "instantiate book attributes correctly" in {
+      assert(game.TheDecagonHouseMurders.attributes === Map(CountryOfOrigin -> Country.Japan, Honkaku -> true, AuthorGender -> GenderEnum.Male,
+        SchoolOrUniversityStudents -> true, DetectiveType -> Detective.Amateur, Series -> true, PushkinVertigo -> true, SuspectPoolType -> SuspectPool.ClosedCircle))
+      assert(game.TheHonjinMurders.title === "The Honjin Murders")
+      assert(game.TheVillageOfEightGraves.attributes(CountryOfOrigin) === Country.Japan)
+      assert(game.TheTokyoZodiacMurders.attributes(Honkaku) === true)
+      assert(game.TheNohMaskMurder.attributes(AuthorGender) === GenderEnum.Male)
+      assert(game.Malice.attributes(SchoolOrUniversityStudents) === false)
+      assert(game.SalvationOfASaint.attributes(DetectiveType) === Detective.PrivateProfessional)
+      assert(game.Hyouka.attributes(Series) === true)
+      assert(game.TheSevenDeathsOfEvelynHardcastle.attributes(PushkinVertigo) === false)
+      assert(game.DanganronpaKirigiri.attributes(SuspectPoolType) === SuspectPool.ClosedCircle)
+    }
+
+    "create a sorted list of all book titles" in {
```

Comment:
is there a reason for them to be sorted?

Reply:
I feel that when printing the remaining books, it's better to ensure that they appear in a clear and consistent order